### DISCUSSION TITLE
FAQs on website - what should they be?

### DISCUSSION THREAD

**Discussion by natalieparellano** [buildpacks/community#84] [Q&A]
Please suggest some FAQs!

Suggestions from [Slack thread](https://buildpacks.slack.com/archives/C94UJCNV6/p1618323281082000):

* Why is my image created 40 years ago?
* why are my env variables not set / why is my binary etc. missing (when not launched through the launcher)
* why is the export/build process slow (with possible pointers to remote publishing, trusted builders etc.)
* why do I get auth errors/docker host issues

**Comment by kvedurmu** [comment]
An explanation of difference between CNB project and CNB implementations (Heroku, Google, Paketo) with links to each would be really helpful
